woocommerce/woocommerce-gutenberg-products-block
# Cart block: show products on backorder

Fixes #2825.

### Screenshots

_Cart:_
![imatge](https://user-images.githubusercontent.com/3616980/87150068-afa44600-c2b1-11ea-8f37-34358f23fab5.png)

_Checkout:_
![imatge](https://user-images.githubusercontent.com/3616980/87150040-9ef3d000-c2b1-11ea-9a0b-cc23400f2420.png)

### How to test the changes in this Pull Request:

1. Edit a product and under Inventory set:
  1.1 Manage stock? Checked.
  1.2 Stock quantity: 0.
  1.3 Allow backorders? Allow, but notify customer.
2. Update the product and go to the frontend of the store.
3. Add that product to the cart.
4. Go to the Cart page and verify the Cart block shows the product is on backorder.
5. Proceed to the Checkout page and verify the backorder notification appears in the sidebar.
6. Would be good to verify there are no style regressions with the 'low stock' and 'product sale' badges.

### Changelog

> Cart and Checkout blocks show a notification for products on backorder.
